legal/prepareDocuments.js

```diff
@@ -0,0 +1,93 @@
+require("babel-register");
```

Comment:
is this duplicated in `commitment` page repo?

Reply:
No - this file is not duplicated in commitment at all.